projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx

```diff
@@ -85,7 +85,7 @@ const FeaturePromptComponent = props => {
 						{ ctaText }
 					</Button>
 					<a href={ nextRoute } onClick={ onDecideLaterClick }>
-						{ __( 'Decide later', 'jetpack' ) }
+						{ __( 'Now now', 'jetpack' ) }
```

Comment:
Did you mean to write "Not now"?

```suggestion
						{ __( 'Not now', 'jetpack' ) }
```

Reply:
Ah yes. my ADHD was causing me some problems last night. :-( Thanks for the review jer.

Reply:
Thank you :)

In the PR description, you said:

> I know this is a massive PR and I should have broken it up into more than one ;-P

Is this PR ready for review now, or will you be adding more things in there? If it's ready for review, do you think you could edit the description and mark it as ready with the labels?